Gemfile

```diff
@@ -18,9 +19,10 @@ gem 'jquery-rails', '~> 3.1.2'
 source 'https://rails-assets.org' do
   gem 'rails-assets-angularjs', '~> 1.3.2'
   gem 'rails-assets-ui-router', '~> 0.2.9'
-  gem 'rails-assets-uuid-js', '~> 0.7.5'
   gem 'rails-assets-checklist-model', '~> 0.2.4'
   gem 'rails-assets-angular-order-object-by', '~> 1.1.1'
+  gem 'rails-assets-text-encoding'
+  gem 'rails-assets-jschardet'
```

Comment:
Please include versions in pessimistic version constraint for these dependencies.